src/core/core.scale.js

```diff
@@ -209,7 +213,12 @@ function parseTickFontOptions(options) {
 	var minor = parseFontOptions(options, options.minor);
 	var major = options.major.enabled ? parseFontOptions(options, options.major) : minor;
 
-	return {minor: minor, major: major};
+	return {
+		minor: minor,
+		major: major,
+		strokeStyle: valueOrDefault(options.strokeStyle, defaults.scale.ticks.strokeStyle),
```

Comment:
These options should probably be included in `parseFontOptions` instead

We have a notion of major & minor ticks, which is best demonstrated here: https://www.chartjs.org/samples/latest/scales/time/financial.html. In that example we have the years as major ticks and have them formatted differently. Putting in `parseFontOptions` will allow the users to have different strokeStyle and lineWidth for major and minor ticks. While I don't have a specific use case in mind for that it would be more consistent with how the other options work

Reply:
Hmm, I think I'm missing something.
It is already in `parseFontOptions`, but if for example, I don't add it here or in `_parseFont`, the arguments are not set, I mean `strokeStyle` and `lineWidth` are empty... where is the "setter"? I don't find it

Reply:
oki! I let the `parseTickFontOptions` how it was and the new options are only in `parseFontOptions`